UnityProject/Assets/Scripts/Items/Tool/FlasherItem.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Objects;
+
+namespace Items.Tool
+{
+	public class FlasherItem : FlasherBase, ICheckedInteractable<HandApply>
+	{
+		public bool WillInteract(HandApply interaction, NetworkSide side)
+		{
+			if (DefaultWillInteract.Default(interaction, side) == false) return false;
+			return gameObject.PickupableOrNull().ItemSlot != null;
+		}
+
+		public void ServerPerformInteraction(HandApply interaction)
+		{
+			if (OnCooldown)
+			{
+				Chat.AddExamineMsg(interaction.Performer,"This flash seems to be on cooldown!");
```

Comment:
Let's make this a little more world-friendly:
```suggestion
				Chat.AddExamineMsg(interaction.Performer,"This flash seems to be recharging!");
```